_includes/evnt-footer.html

```diff
@@ -10,15 +10,15 @@
 <h2>Corporate Supporters</h2>
 <div id="corp_member_div">
   {% for member in corpsupporters %}
-  <a href="{{member.url}}" class="member-logo" rel="sponsored" onclick='handleOutboundLinkClicks(event);' target="_blank"><img src="{{member.logo}}" alt="image"/></a>
+  <a href="{{member.url}}" class="member-logo" rel="sponsored" onclick='handleOutboundLinkClicks(event);' target="_blank" rel="noopener noreferrer"><img src="{{member.logo}}" alt="image"/></a>
```

Comment:
```suggestion
  <a href="{{member.url}}" class="member-logo" onclick='handleOutboundLinkClicks(event);' target="_blank" rel="sponsored noopener noreferrer"><img src="{{member.logo}}" alt="image"/></a>
```

As Harold suggested.